### ISSUE TITLE
Desktop GTX 1080Ti is 20X  slower than 2070 laptop

### ISSUE THREAD

**Issue by RodTDai** [lllyasviel/Fooocus#883]
Hey guys.

I have on my linux PC, a 1080Ti, and a 2070 on my Win11 aptop.

In 1080Ti is running like 20x slower, why? Maybe is related to tensorRT?
is is not supposed to be so slow on 1080Ti, as Automatic1111 is at almost the same speed in both machines when not using tensorRT.

Maybe some issue is causing 1080Ti to use optimization not appropriated to it and it is doing the oposite effect?

Thanks.

Follow the logs for the 1080Ti on Linux Mint:

$ python3.10 entry_with_update.py 

Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py']
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
Fooocus version: 2.1.780
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Total VRAM 11169 MB, total RAM 64141 MB
Set vram state to: NORMAL_VRAM
Disabling smart memory management
Device: cuda:0 NVIDIA GeForce GTX 1080 Ti : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
missing {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: /media/rod/4TDATA/AI_TOOLS/FOOOCUS/Fooocus/models/checkpoints/juggernautXL_version6Rundiffusion.safetensors
LoRAs loaded: [('sd_xl_offset_example-lora_1.0.safetensors', 0.1), ('None', 0.1), ('None', 0.1), ('None', 0.1), ('None', 0.1)]
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.35 seconds
App started successful. Use the app with http://127.0.0.1:7860/ or 127.0.0.1:7860
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 7973290471544724091
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 24
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] racoon, vivid colors, ambient light, dynamic background, highly detailed, elegant, professional fine detail, sharp focus, perfect composition, dramatic cinematic full, crisp quality, extremely aesthetic, attractive, intricate, innocent, worthy, rich, modern, futuristic, stunning, epic, gorgeous, amazing, marvelous, breathtaking, novel, creative, beautiful, awesome, cool, very inspirational
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] racoon, highly detailed, dramatic light, sharp focus, excellent composition, dynamic, aesthetic, very inspirational, appealing, expressive, epic, stunning, brilliant, cinematic, fine detail, clear, color, complex, amazing, awesome, elegant, intricate, intense, artistic, thought, imposing, professional, best, winning, decorated, iconic, creative, perfect, trendy
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.13 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
Preparation time: 2.82 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.02916753850877285, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.41 seconds
  3%|██                                                          | 1/30 [00:19<09:33, 19.77s/it]
User stopped
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
Total time: 139.35 seconds
[Fooocus Model Management] Moving model(s) has taken 0.54 seconds

**Comment by RodTDai** [comment]
i also tried to move to SSD, but still taking 5-6 minutes instead of 30-40 seconds

**Comment by SaFaUU** [comment]
Yes. It takes a lot of time and gets stuck in adm 2816 line in the console.

**Comment by mashb1t** [comment]
See https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md for a HowTo set up swap. Known working afterwards.
